OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -195,6 +197,8 @@ public DefaultSpriteSequence(ModData modData, string tileSet, SpriteCache cache,
 				var flipY = LoadField(d, "FlipY", false);
 
 				Facings = LoadField(d, "Facings", 1);
+				InterpolateFacings = LoadField(d, "InterpolateIntermediateFacings", false);
```

Comment:
Thinking about this more, I now disagree with my original advice for two reasons:
* Sprites distort in an ugly way when rotated by a small amount, and continuous rotation looks odd when combined with quantized rotation from other sprites.
* There are several parts of the code (e.g. `BodyOrientation`, `Turreted`) that rely on the sequence facing number for quantizing simulation-related rotations. These will be broken by the current implementation.

A better solution would be to define an `InterpolatedFacings: <integer>` field, have `GetSprite` quantize to the nearest interpolated facing, and introduce a new method that `BodyOrientation` etc can call to return the appropriate facing value depending on how the sequence is defined.

`UseRotationsForFacings` should then be redundant against defining `Facings: 1` with e.g. `InterpolatedFacings: 32`.

Reply:
Good idea, in the new version I now have the one `InterpolatedFacings` attribute, which works as you describe, by quantizing the original rotation from `AngleDiffToStep()` to the nearest interpolated facing. In this way `Facings: 16, InterpolatedFacing: 32` will display 32 facings, 16 of which come from the original `Facing` attribute, while the remaining 16 are created by applying rotation. Then to get the same behaviour as shown in my original PR, you simply set `InterpolateIntermediateFacings: 1024`, which I have done in the latest version. This works because 1024 facings makes the step size 1, which prevents the quantizing from reducing the number of rotations. Also this now can be used in place of `UseRotationsForFacings` just as you describe, by simply setting `Facings: 1` and `InterpolatedFacings: n` where n is however many interpolated facings you want. Have tested this and it seems to work, with one small caveat, that having a Facing of 1 does not seem to be compatible with the `UseClassicSprites` flag due to it requiring 32 facings. But I suspect that isn't important since the UseClassicSprites flag becomes unnecessary if only one facing is used.